package.json

```diff
@@ -1,125 +1,125 @@
-{

```

Comment:
I think everything shows to be replaced here due to line-ending change from CRLF to LF.

The only actual functional change is here:
```
  "exports": {
    ".": {
      "node": "./lib.cjs/index.native.js",
```
to 
```
  "exports": {
    ".": {
      "node": "./lib.cjs/index.js",
```

(no `native` varianta anymore)